Automattic/wp-calypso
# Site Profiler: fix white space issue on domain input

Related to #82819

## Proposed Changes

* Add a new trimDomain function to clean up the white spaces from the user input

## Testing Instructions



* Navigate to http://calypso.localhost:3000/site-profiler
* Type in a domain with a white space in it, for example, `examp le.com`
* See if it shows the validation error message
* Try different domains and make sure we don't break anything

## Pre-merge Checklist
